symbol_exporter/ast_db_populator.py

```diff
@@ -39,9 +39,13 @@ def parse_so(filename):
     return c_symbols_to_datamodel(CompiledPythonLib(filename).find_symbols())
 
 
-def single_so_file_extraction(so_file):
+def single_so_file_extraction(so_file, top_dir):
```

Comment:
shouldn't this logic belong to the .so parsing module itself - seems a concern of the .so parser and not the db_populator?

Reply:
That's fair, really this is part of the SO parser's equivalent of package handling.